static/js/people.js

```diff
@@ -704,6 +706,7 @@ exports.filter_all_users = function (pred) {
     return ret;
 };
 
+// TODO: change function to receive is_active.
```

Comment:
I think here we could have something like `exports.get_realm_users = function (is_active)` and the function below be `exports.get_humans = function(is_active)` because there is no need to make another function that performs the same actions but only in another data structure, `non_active_user_dict`. I'll do another commit with these changes.

Reply:
I don't think we need one function for this.  Passing in booleans makes the calling code hard to read.  I'd just keep the functions close together and have similar names.

Reply:
hmm ok, then.